src/main/java/org/folio/settings/server/storage/SettingsStorage.java

```diff
@@ -143,10 +143,10 @@ static List<String> getCqlLimitPermissions(
         if (rights.contains(PERM_USERS)) {
           queryLimits.add(scopeEq);
         } else if (rights.contains(PERM_OWNER) && currentUser != null) {
-          queryLimits.add("(" + scopeEq
-              + " and (userId <> \"\" or userId == \"" + currentUser + "\"))");
+          queryLimits.add("(" + scopeEq + " not userId = \"\")");
+          queryLimits.add("(" + scopeEq + " and userId == \"" + currentUser + "\")");
         } else {
-          queryLimits.add("(" + scopeEq + " and userId <> \"\")");
+          queryLimits.add("(" + scopeEq + " not userId = \"\")");
```

Comment:
Just for my interest ... is the change from `and userId <> ""` to `not userId = ""` actually changing behaviour (because these are special cases) or is it merely rephrasing?

Should `=` be `==` here?

Not part of the present PR, but it would be good if the folio-vertx-lib README explained any special cases such as RMB's weird ways as asking "does this field exist in the record"?

Reply:
Only `= ""` is special, matching all fields that are NOT NULL. Same behavior as RMB.

Until now folio-vertx-lib had special handling of <> "" for NULL, but that's now gone and the handling is simlar to RMB in this respect.

Let me know if you think we should express this in another way in CQL with a relation modifier for example.

Reply:
I do think `= ""` matching non-empty values is pretty insane; but we've lived with it for now, and I think there is value in doing this the same way as in RMB.

Does `field == ""` search for actual empty fields?

Reply:
Yes. == "" searches for empty string.